test/pylib/manager_client.py

```diff
@@ -113,34 +113,34 @@ async def mark_dirty(self) -> None:
         await self._request("/cluster/mark-dirty")
 
     async def server_stop(self, server_id: str) -> bool:
-        """Stop specified node"""
-        ret = await self._request(f"/cluster/node/{server_id}/stop")
+        """Stop specified server"""
```

Comment:
I am getting a sense of deja-vu. Didn't I already commit this patch? If I did, please rebase this PR.